src/Domain/Model/DirectDebitPayment.php

```diff
@@ -5,25 +5,19 @@
 namespace WMDE\Fundraising\PaymentContext\Domain\Model;
 
 use DateTimeImmutable;
+use WMDE\Euro\Euro;
 
-/**
- * @license GPL-2.0-or-later
- * @author Jeroen De Dauw < jeroendedauw@gmail.com >
- */
-class DirectDebitPayment implements PaymentMethod {
+class DirectDebitPayment extends Payment {
 
-	private BankData $bankData;
+	private const PAYMENT_METHOD = 'BEZ';
 
-	public function __construct( BankData $bankData ) {
-		$this->bankData = $bankData;
-	}
-
-	public function getId(): string {
-		return PaymentMethod::DIRECT_DEBIT;
-	}
+	private ?Iban $iban;
```

Comment:
While reviewing this, I had to think about why this is nullable. To make this more explicit, I suggest the following changes: